modules/standard/CommDiagnostics.chpl

```diff
@@ -17,86 +17,269 @@
  * limitations under the License.
  */
 
-// CommDiagnostics.chpl
-//
-pragma "no use ChapelStandard"
+/* This module provides for collecting and reporting statistics about
```

Comment:
No, I actually did intend "provides for", as in "a well-formed spout provides for tidy pouring". But I can see how that might be unclear. I think I'll change it to "provides for the collection of statistics", since reporting actually isn't covered by the module. Would that be clearer?